packages/react/src/primitives/Select/__tests__/Select.test.tsx

```diff
@@ -0,0 +1,203 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { Select } from '../Select';
+import { IconExpandMore } from '../../Icon';
+import { ComponentClassNames } from '../../shared';
+
+describe('Select primitive test suite', () => {
```

Comment:
Can you please add a test that shows this select component can be [controlled](https://reactjs.org/docs/forms.html#the-select-tag)?

Reply:
Yup! Will do